src/components/ChartFooter.jsx

```diff
@@ -5,19 +5,22 @@ import Tooltip from './Tooltip.jsx';
 import ChartSetting from './ChartSetting.jsx';
 import '../../sass/components/chart-footer.scss';
 
+const ChartFooterNetwork = React.memo(({ networkStatus }) => (
+    <Tooltip
+        className="sc-chart-footer__item sc-chart-footer__item--status"
+        content={`${t.translate('Network status')}: ${networkStatus ? (t.translate(networkStatus.tooltip)) : ''}`}
+        enabled
+    >
+        <div className={`sc-chart__status sc-chart__status--${networkStatus ? networkStatus.class : 'offline'}`} />
+    </Tooltip>
+));
```

Comment:
by doing this, I prevent `Tooltip` component to render on each server time update